vcf/model.py

```diff
@@ -117,6 +117,22 @@ def is_het(self):
             return None
         return self.gt_type == 1
 
+    @property
+    def is_filt(self):
+        """ Return True for filtered calls """
+        try: # no FT annotation present for this variant
+            FT=self.data.FT
+        except AttributeError:
+            return False
+        if FT == None or FT == []: # FT is not set or set to PASS
+            return False
+        elif len(FT) > 0: # FT contains one or more filters
+            return True
+        else: # This should not happen
```

Comment:
I don't think it is needed to add special casing for things that can never happen. I'd just have a default `return True` or `return False` as the default case.